classes/core/PKPApplication.php

```diff
@@ -170,6 +170,7 @@ class_alias('\PKP\payment\QueuedPayment', '\QueuedPayment'); // QueuedPayment in
         }
 
         // Ensure that nobody registers for hooks that are no longer supported
+        Hook::addUnsupportedHooks('API::_submissions::params', 'Template::Workflow::Publication', 'Template::Workflow', 'Workflow::Recommendations'); // pkp/pkp-lib#10766 Removed with new submission lists for 3.5.0
```

Comment:
@Vitaliy-1 Just wondering, should not we add hooks for the new _submissions endpoints?